jenkins/README.md

```diff
@@ -45,6 +45,15 @@ If the author is not in the whitelist but should be trusted then by adding a
 comment **add to whitelist** on the PR, the author will then be able to run the
 jobs on its own.
 
+### How to clean up leftover VMs from CityCloud
+There is a Jenkins [master job](https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup/)
+that every 6 hours cleans up all the leftover VMs, which fail to be deleted
+at the end of v1alphaX integration test.
+
+**Note:** If you want to trigger cleaning job manually, you can use the
+`/test-clean` phrase within an open pull request under
+[metal3-io/project-infra](https://github.com/metal3-io/project-infra) repo.
```

Comment:
The purpose is also to be able to test a change in the cleaning script. To run it manually, I would rather recommend to do it through the Jenkins Web UI

Reply:
Does everyone have an access to the Jenkins UI to do so? or should I skip this part here?